pkg/reconciler/instances/eventing/preaction/removenatsoperatorstep_test.go

```diff
@@ -0,0 +1,76 @@
+package preaction
+
+import (
+	"context"
+	pmock "github.com/kyma-incubator/reconciler/pkg/reconciler/chart/mocks"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/kubernetes"
+	"github.com/stretchr/testify/mock"
+	"go.uber.org/zap"
+	"k8s.io/client-go/kubernetes/fake"
+	"testing"
+
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/kubernetes/mocks"
+	"github.com/stretchr/testify/require"
+
+	"github.com/kyma-incubator/reconciler/pkg/logger"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/chart"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/service"
+)
+
+func TestDeletingNatsOperatorResources(t *testing.T) {
```

Comment:
Could you please add a test-case to simulate a 2.X cluster which does not have 1.X resources?